src/main/java/de/qabel/desktop/ui/feedback/FeedbackController.java

```diff
@@ -47,19 +59,27 @@ public void run() {
                 try {
                     reportHandler.sendFeedback(feedbackField.getText(), nameField.getText(), emailField.getText());
 
-                    feedbackField.setText("");
-                    nameField.setText("");
-                    emailField.setText("");
-                } catch (IOException  e) {
+                    Platform.runLater(() -> {
+                        createInfoBox(infoMessage, titleBar);
+                        feedbackField.setText("");
+                        nameField.setText("");
+                        emailField.setText("");
+                    });
+
+
+                } catch (IOException e) {
                     alert(e);
                 }
             }
-        }.start();
-
-
-
 
+        }.start();
     }
-}
 
+    void createInfoBox(String infoMessage, String titleBar) {
```

Comment:
you don't need to fetch those strings from the resourceBundle on init. I expect the number of feedbacks sent to be much lower than the times the form gets opened, so you should deplay the resourceBundle.getString() to when the dialog actually opens